tests/saputils_test.py

```diff
@@ -0,0 +1,112 @@
+"""
+Unitary tests for saputils.py.
+
+:author: sisingh
+:organization: SUSE LLC
+:contact: sisingh@suse.com
+
+:since: 2020-03-26
+"""
+
+# pylint:disable=C0103,C0111,W0212,W0611
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import logging
+import unittest
+import filecmp
+import shutil
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+from shaptools import saputils
+
+class TestSapUtils(unittest.TestCase):
+    """
+    Unitary tests for shaptools/saputis.py.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Global setUp.
+        """
+
+        logging.basicConfig(level=logging.INFO)
+
+    def setUp(self):
+        """
+        Test setUp.
+        """
+
+    def tearDown(self):
+        """
+        Test tearDown.
+        """
+
+    @classmethod
+    def tearDownClass(cls):
+        """
+        Global tearDown.
+        """
+
+    @mock.patch('shaptools.shell.execute_cmd')
+    @mock.patch('os.path.isfile')
+    def test_extract_sapcar_file(self, mock_sapcar_file, mock_execute_cmd):
```

Comment:
Please, test the `output_dir` and `options` entries.

Reply:
Thanks, I added those options now